static/app/views/organizationGroupDetails/quickTrace/configureDistributedTracing.tsx

```diff
@@ -192,7 +192,7 @@ const ExampleQuickTracePanel = styled(Panel)`
   gap: ${space(1)};
   background: none;
   padding: ${space(2)};
-  margin: ${space(2)} 0;
+  margin: ${space(2)} 0 0;
```

Comment:
should this just be `margin-top` then?

Reply:
`Panel` has its own set of margins, so we need to reset it here too.